reader for illustris simulations

LihanGuo opened the pull request:
[Reader] :apostle (for illustris simulations)

LihanGuo committed:
Add files via upload

LihanGuo committed:
Add files via upload

Kambrian commented:
this is in conflict with the existing reader. can you rewrite to extend the existing code instead of overwritting it?

LihanGuo committed:
an extended version

I rewrite the previous version by adding a function called Whetherillustris to extend the existing code instead of overwritting it.

LihanGuo committed:
Eliminate differences of the blank space

LihanGuo committed:
Eliminate differences of the blank space

LihanGuo committed:
Update datatypes.h

LihanGuo committed:
Delete datatypes.h

LihanGuo committed:
Add files via upload

LihanGuo committed:
Add files via upload

LihanGuo committed:
latest

LihanGuo committed:
remove new datatype to local function

LihanGuo committed:
Name the variables more canonically
